test/seed/852-rule.rule.merge.yaml

```diff
@@ -165,7 +199,6 @@ splitName: # defaults name: fee, tag: |fee|acquirer|
   - {conditionName: Test by start date, name: base, tag: "|issuer|"}
   - {conditionName: Test by end date, name: testFee}
   - {conditionName: Test by end date, name: base, tag: "|issuer|"}
-  - {conditionName: Test settlement, name: base, tag: "|issuer|"}
```

Comment:
should we remove this rule?

Reply:
It was removed by mistake while resolving merge conflicts. I have added it again.